index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@100&family=Noto+Sans+JP&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+    <header>My Logo</header>
+    <div class="menu-img"><img src="images/Union.svg"></div>
+    </div>
+    <section id="first-sec">
+        <h1>I'm Tebe <br> Glad to see <br> you!</h1>
```

Comment:
- [ ] The Font-family used here is at variance from what is expected which is "Poppins". Please ensure that you use the correct font type across this project with the same provided in the Figma design. 

Your design | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/219355680-83a7ef3e-0b0b-42fe-8fd8-4b83976f687f.png) | ![image](https://user-images.githubusercontent.com/84156714/219355776-ea47176f-b5c7-4518-9a34-f9b2960c10c0.png)